api/server.py

```diff
@@ -1,54 +1,62 @@
 #!/usr/bin/env python
-#
-# ATD - CR3 Download API
-#
-
+"""
+API which handles
+- Auth0 user management
+- CR3 PDF downloads
+- Person image uploads and downloads
+"""
 import datetime
 import json
-import os
+from os import getenv, getpid, sysconf_names, sysconf
 import re
 import secrets
 import string
-import sys
 
 import boto3
-import requests
-
 from dotenv import load_dotenv, find_dotenv
-from os import environ as env
-from functools import wraps
-from six.moves.urllib.request import urlopen
-
 from flask import Flask, request, jsonify, g
 from flask_cors import cross_origin
-from werkzeug.local import LocalProxy
+from functools import wraps
 from jose import jwt
+from PIL import Image
+import requests
+from six.moves.urllib.request import urlopen
+from werkzeug.local import LocalProxy
+
+from utils.graphql import (
+    make_hasura_request,
+    GET_PERSON_IMAGE_METADATA,
+    UPDATE_PERSON_IMAGE_METADATA,
+)
 
-#
-# Environment
-#
 ENV_FILE = find_dotenv()
 if ENV_FILE:
     load_dotenv(ENV_FILE, verbose=True)
 
-# We need the Auth0 domain, Client ID and current api environment.
-AUTH0_DOMAIN = os.getenv("AUTH0_DOMAIN", "")
-CLIENT_ID = os.getenv("CLIENT_ID", "")
-API_CLIENT_ID = os.getenv("API_CLIENT_ID", "")
-API_CLIENT_SECRET = os.getenv("API_CLIENT_SECRET", "")
-
-# AWS Configuration
-AWS_DEFAULT_REGION = os.getenv("AWS_DEFAULT_REGION", "us-east-1")
-AWS_S3_KEY = os.getenv("AWS_S3_KEY", "")
-AWS_S3_SECRET = os.getenv("AWS_S3_SECRET", "")
-AWS_S3_CR3_LOCATION = os.getenv("AWS_S3_CR3_LOCATION", "")
-AWS_S3_BUCKET = os.getenv("AWS_S3_BUCKET", "")
+AUTH0_DOMAIN = getenv("AUTH0_DOMAIN", "")
+CLIENT_ID = getenv("CLIENT_ID", "")
+API_CLIENT_ID = getenv("API_CLIENT_ID", "")
+API_CLIENT_SECRET = getenv("API_CLIENT_SECRET", "")
+AWS_DEFAULT_REGION = getenv("AWS_DEFAULT_REGION", "us-east-1")
+AWS_S3_KEY = getenv("AWS_S3_KEY", "")
+AWS_S3_SECRET = getenv("AWS_S3_SECRET", "")
+AWS_S3_BUCKET_ENV = getenv("AWS_S3_BUCKET_ENV", "")
+AWS_S3_CR3_LOCATION = f"{AWS_S3_BUCKET_ENV}/cr3s/pdfs"
+AWS_S3_PERSON_IMAGE_LOCATION = f"{AWS_S3_BUCKET_ENV}/images/person"
+AWS_S3_BUCKET = getenv("AWS_S3_BUCKET", "")
```

Comment:
i reworked a few of these env vars and updated our 1pass entry accordingly. we're going to need to bring the ECS environments in sync with these changes